plugins/woocommerce/src/Internal/Admin/Reviews.php

```diff
@@ -74,6 +76,98 @@ public function add_reviews_page() {
 		add_action( "load-{$this->reviews_page_hook}", [ $this, 'load_reviews_screen' ] );
 	}
 
+	/**
+	 * Determines whether the current page is the reviews page.
+	 *
+	 * @return bool
+	 */
+	public function is_reviews_page() : bool {
+		global $pagenow;
+
+		return 'edit.php' === $pagenow && isset( $_GET['page'] ) && 'product-reviews' === $_GET['page'];
+	}
+
+	/**
+	 * Displays notices on the Reviews page.
+	 *
+	 * @return void
+	 */
+	public function display_notices() {
+
+		if ( $this->is_reviews_page() ) {
+			$this->maybe_display_reviews_bulk_action_notice();
+		}
+	}
+
+	/**
+	 * May display the bulk action admin notice.
+	 *
+	 * @return void
+	 */
+	protected function maybe_display_reviews_bulk_action_notice() {
+
+		$messages = $this->get_bulk_action_notice_messages();
+
+		echo ! empty( $messages ) ? '<div id="moderated" class="updated"><p>' . implode( "<br/>\n", $messages ) . '</p></div>' : '';  // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+	}
+
+	/**
+	 * Gets the applicable bulk action admin notice messages.
+	 *
+	 * @return array
+	 */
+	protected function get_bulk_action_notice_messages() : array {
+
+		$approved   = isset( $_REQUEST['approved'] ) ? (int) $_REQUEST['approved'] : 0;
+		$unapproved = isset( $_REQUEST['unapproved'] ) ? (int) $_REQUEST['unapproved'] : 0;
+		$deleted    = isset( $_REQUEST['deleted'] ) ? (int) $_REQUEST['deleted'] : 0;
+		$trashed    = isset( $_REQUEST['trashed'] ) ? (int) $_REQUEST['trashed'] : 0;
+		$untrashed  = isset( $_REQUEST['untrashed'] ) ? (int) $_REQUEST['untrashed'] : 0;
+		$spammed    = isset( $_REQUEST['spammed'] ) ? (int) $_REQUEST['spammed'] : 0;
+		$unspammed  = isset( $_REQUEST['unspammed'] ) ? (int) $_REQUEST['unspammed'] : 0;
+
+		$messages = [];
+
+		if ( $approved > 0 ) {
+			/* translators: %s is an integer higher than 0 (1, 2, 3...) */
+			$messages[] = sprintf( _n( '%s comment approved', '%s comments approved', $approved, 'woocommerce' ), $approved );
```

Comment:
```suggestion
			$messages[] = sprintf( _n( '%s review approved', '%s reviews approved', $approved, 'woocommerce' ), $approved );
```

Reply:
Good call! Fixed in https://github.com/godaddy-wordpress/woocommerce/pull/34/commits/bff3187ef8c2dd2c77b63a90b4d852134267e49a.